agent/src/agent/pipeline/pipeline.py

```diff
@@ -150,7 +150,7 @@ def timestamp_format(self) -> str:
 
     @property
     def values(self) -> list:
-        if self.source.type == source.TYPE_INFLUX:
+        if self.source.type == 'influx':
```

Comment:
why don't use constants anymore?

Reply:
I removed source because of circular imports, I didn't try to fix it in this pr in order not to make it bigger, I'll leave a todo here, we can discuss this problem